src/utils/axios.ts

```diff
@@ -0,0 +1,37 @@
+import axios from "axios";
+import _ from "lodash";
```

Comment:
Do you think we need the whole lodash lib as you import it all. 
Just asking.

Reply:
I don't think we need to import the whole lib.
How about I change it to only import the necessary tool from 'lodash-es'

Reply:
That's would be way better.